### ISSUE TITLE
Accessibility - iOS - French - Header/Top Banner Read in English Instead of French 

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#103]
All the header/top banner items (ILAB logo, application name, and DOL seal) are read as English words along with the search button and the information button on the bottom of the home screen and other screens instead of the chosen language. Those should be translated into the language the person has chosen and not English. Some examples are ILAB (Bureau of International Labor Affairs) needs to be translated into French along with "United States Department of Labor" seal, search button, information button, etc.

**Comment by sanganalakshay** [comment]
fixed

**Comment by jfc3-dol** [comment]
I don't speak French, but the header section is not speaking English. So it seems that the language is set correctly and the header information appears to have been translated accurately.

Closing issue.
